### PR TITLE
Project 18 - Caroline Fahlgren 

### PR DESCRIPTION
**Pull request by CarolineF3**
https://mongo-api-carolinef.herokuapp.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Technologies: Mongo DB, Mongoose, API, Express, Javascript ES6, Node.js
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://mongo-api-carolinef.herokuapp.com/
```
Comment (Pauan8) [inline README.md:11]: I could not get any data from the endpoints. It all said "something went wrong"

File: server.js [hunk 2]
```diff
@@ -1,39 +1,114 @@
...
+  const authorRegex = new RegExp(author, "i");
+  const titleRegex = new RegExp(title, "i");
+
+  try {
+    const books = await Book.find({
+      authors: authorRegex,
+      title: titleRegex,
+    })
+      .skip(page * 10)
+      .limit(10);
```
Comment (Pauan8) [inline server.js:66]: Nice pagination :)

File: server.js [hunk 3]
```diff
@@ -1,39 +1,114 @@
...
+
+  try {
+    const singleBook = await Book.findById(bookID);
+    if (singleBook) {
+      res.status(200).json(singleBook);
+    } else {
+      res.status(404).json({ error: "No books with this ID" });
+    }
+  } catch (error) {
+    res.status(400).json({ error: "Something went wrong", details: error });
```
Comment (Pauan8) [inline server.js:85]: This error line is used a lot, it could be an idea to put it in a function and call it when needed ^^

File: data/server.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  //v2 - classic .then()
+  // Member.findOne({ name: memberName })
+  // .then(data => res.json(data))
+  // .catch(error => res.status(400).json({ error: 'Something went wrong', details: error }))
+});
+
+app.listen(port, () => {
+  // eslint-disable-next-line
+  console.log(`Server running on http://localhost:${port}`);
+});
```
Comment (Pauan8) [inline data/server.js:124]: What is this file doing in this project?

**Review by Pauan8 — APPROVED** [review]
A nice well structured project! 💯 
Unfortunatly the employed heroku API didn't work for me, so you should look at that.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please check your DB connection because your API returns error all the time 😭

**Review by KarolCislowski — APPROVED** [review]
Nice API!

**Pull request closed without merging**